### DISCUSSION TITLE
How to configure loop & slidesPerView in SwiperV9

### DISCUSSION THREAD

**Discussion by redd97** [nolimits4web/swiper#6444] [Q&A]
I used Swiper version 8.4.7, and found a [bug](https://github.com/nolimits4web/swiper/issues/6435). 
But since it is not supported, I tried to upgrade to version 9
But I, and not only ran into problems( [1](https://github.com/nolimits4web/swiper/issues/6440) , [2](https://github.com/nolimits4web/swiper/issues/6424) , [3](https://github.com/nolimits4web/swiper/issues/6397) , [4](https://github.com/nolimits4web/swiper/issues/6395) , [5](https://github.com/nolimits4web/swiper/issues/6372) , [and similar](https://github.com/nolimits4web/swiper/issues/6419) ) with slidesPerView and loop when trying to make a responsive slider using breakpoints

Since the 'loop' of the new version of the slider has a limitation (number of slides must be >= slidesPerView * 2), I'm stumped on how to make a looped slider.
For example:
I have four slides.
On the phone it shows 1.  (slidesPerView * 2 <= 4) Should work
On the tablet 2.    (slidesPerView *2 <= 4 ) Should work
On the computer 3.  (slidesPerView *2 > 4) will break

The idea of duplicating slides comes to mind, but there will already be problems with pagination:
 - increasing the number of dots
 - switch to slide an set it's active when you click on the dot (slide to original slide, or duplicate) and more.

How can I solve this problem?

**Comment by mamlzy** [comment]
i'm facing the same issue, is there any updates?
